docs/getting-started/overview.rst

```diff
@@ -1,6 +1,113 @@
-.. include:: /stub-topic.txt
+Overview of .NET implementations
+================================
+
+.NET Platform is, at its very core, a set of standards that can be implemented. There are 
+various implementations, some coming from Microsoft, some coming from other companies and 
+groups. This document will outline the current state of the .NET ecosystem and its various 
+implementations, so you can get a good "lay of the land". 
```

Comment:
"lay of the land" is a bit of a dated colloquialism. How about just delete that whole last sentence?